app/assets/stylesheets/donor.css.scss

```diff
@@ -208,20 +212,20 @@
 
 .donation-item-canceled {
   border-left: 10px solid $donation-canceled-key;
+  .donation-status p strong {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 4